apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/source/ClaimTransformerFieldTester.java

```diff
@@ -44,7 +44,6 @@
    *     entity object
    * @param fieldLabel text identifying the field in {@link DataTransformer.TransformationException
    *     error messages}
-   * @param minLength minimum valid length for the string field
```

Comment:
Interesting to see the `minLength` dropped out of these!

Reply:
Was removed from the test at some point at the test, this is just fixing the docs to match. Not sure behind the actual reason for the original adjustment though.